index.html

```diff
@@ -1725,6 +1726,69 @@ <h3>Validity Period</h3>
 }
         </pre>
 
+        <h4>Representing Time</h4>
+
+        <p>
+When representing time values, implementers are urged to understand that times
+are not implemented in a uniform manner throughout the world. For example:
+        </p>
+
+        <ul>
+          <li>
+The observation of daylight savings time does not start or end on the same day
+and time, if at all, between various regions.
+          </li>
+          <li>
+The adoption of the Gregorian calendar did not happen in a single year; it was
+phased in over many years across various societies.
+          </li>
+          <li>
+Leap seconds might not be taken into account in all software systems, especially
+for dates and times that precede the introduction of the leap second.
+          </li>
+        </ul>
+
+        <p>
+These are just a few examples that illustrate that whether or not a time is seen
+as valid can be based on cultural norms. In other words, some dates and times
+exist for some cultures, while they do not exist for other cultures.
+        </p>
+
+        <p>
+This specification attempts to increase the number of universally recognized
+combinations of dates and times, and reduce the potential for
+misinterpretation of time values, by utilizing the
+<code>dateTimeStamp</code> construction first established by the [<a
+data-cite="XMLSCHEMA11-2#dateTimeStamp">XMLSCHEMA11-2</a>] specification. In
+order to reduce misinterpretations between different cultures, all times are
+specified as offsets against Universal Coordinated Time (UTC).
+        </p>
+
+        <p>
+Implementers that desire to check <code>dateTimeStamp</code> values for validity
```

Comment:
Why copy the regex here instead of [linking](https://www.w3.org/TR/xmlschema11-2/#dateTime-lexical-mapping) to it?  How to interpret time correctly seems out of place in a spec like this.

Reply:
Because the issue this PR is trying to address specifically requested that the Regex be copied into the spec: https://github.com/w3c/vc-data-model/issues/846 

I agree that linking to it is probably better, but that would mean that implementers would have to interpret the way XML Schema 1.1 Part 2 layers regexes to achieve the regex that's provided in this PR.

If there is somewhere else that we could point to that highlights the issues regarding interpretation of times, let me know. I couldn't find a good place to point to that had a summary like this.

Reply:
I think copying other spec details is unneeded, error prone, and bloats this spec.  If you do insist on including it, at least add the RFC note about removing whitespace in the regex.

Reply:
Fixed in 555d69fcb5f7289f9474a7bc541d59f59cfbf9ac.